docs/writing-reactors/distributed-execution.mdx

```diff
@@ -149,22 +116,12 @@ Upon running the program, you will see information printed about the starting an
 
 You may have several federations running on the same machine(s) or even several instances of the same federation. In this case, it is necessary to distinguish between the federations. To accomplish this, you can pass a `-i` or `--id` parameter to the RTI and its federates with an identifier that is unique to the particular federation. For example,
 <ShowIfs>
-  <ShowIf c>
+  <ShowIf c py>
 
   ```sh
   RTI -n 2 -i myFederation
-  bin/Federated_s -i myFederation
-  bin/Federated_d -i myFederation
-  ```
-
-  </ShowIf>
-
-  <ShowIf py>
-
-  ```sh
-  RTI -n 2 -i myFederation
-  python3 src-gen/Federated/s/Federated_s.py -i myFederation
-  python3 src-gen/Federated/d/Federated_d.py -i myFederation
+  fed-gen/Federated/bin/federated__s -i myFederation
```

Comment:
Shouldn't the RTI line above (122) also be updated?

Reply:
Yes, that is right. Great catch, thanks!